AutomatedTesting/Levels/auto_test/auto_test.prefab

```diff
@@ -0,0 +1,214 @@
+{
+    "ContainerEntity": {
+        "Id": "ContainerEntity",
+        "Name": "auto_test",
```

Comment:
nit: The name is a bit generic. Can we rename this prefab file to something specific?Looks like it has some script components that require some assets. So maybe rename to something that  indicates that? 
Or will that require a lot of refactoring in other tests?

Reply:
I am not sure if that is a good idea seems that some other AP tests are using auto_test as well.

Reply:
We shouldn't be afraid to rename files/functions just because the name is being used in other places too. Particularly the generic names or names that don't reveal anything won't help the developers in anyway. Since this is a pre-existing issue, we can push these changes for now but please create a follow up issue to rename this to something better.

Reply:
Yeah I was thinking to create tickets for these nits as well.

Reply:
Follow-up: #7504